tests/data/groundtruth/docling_v2/code_and_formula.md

```diff
@@ -1,15 +1,21 @@
-## Java Code Example
+1 2 3 4
+
+```
+function add(a, b) { return a + b; } console.log(add(3, 5));
+```
```

Comment:
do we understand why this is on top of the page?
is there an issue with reading order caused by the line numbers?